### ISSUE TITLE
ToolbarFlex: Loosing buttons

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise#7421]
**Describe the bug**
Some problem happened in the release that is causing buttons to be lost. Can be seen in a couple ways.

**To Reproduce**
1. Go to https://main-enterprise.demo.design.infor.com/components/editor/example-index.html
2. Resize the page less wide (about 25%ish size and sto )
4. See error -> buttons fit but they are not visible

1. Open https://github.com/infor-design/enterprise-ng and start the app
2. http://localhost:4200/ids-enterprise-ng-demo/
3. look for the ...  (personalization) button and its missing

**Additional context**
Seems ok in previous versions https://4803-enterprise.demo.design.infor.com/components/editor/example-index.html

**Comment by tmcconechy** [comment]
@andriivarhanov i did notice that this code is newer https://github.com/infor-design/enterprise/blob/main/src/components/toolbar-flex/toolbar-flex.js#L495-L501 maybe have a look there.

Also possibly add a guard on that code as per https://teams.microsoft.com/l/message/19:6d4c9079385840c0bc61e55bcdd16a33@thread.skype/1681479872996?tenantId=457d5685-0467-4d05-b23b-8f817adda47c&groupId=4f50ef7d-e88d-4ccb-98ca-65f26e57fe35&parentMessageId=1681479872996&teamName=IDS%20Enterprise%20Development&channelName=Problems&createdTime=1681479872996

**Comment by andriivarhanov** [comment]
@tmcconechy yeah, I found it, the code is added for datagrid contextual toolbar flex buttons https://github.com/infor-design/enterprise/pull/7364 , I'm thinking I'm isolating it for the contextual toolbar only, just a few checks and PR should be ready later today
